### DISCUSSION TITLE
Train a subject face in the Dreambooth extension using any SD v2.1

### DISCUSSION THREAD

**Discussion by TypicalPlay** [d8ahazard/sd_dreambooth_extension#546] [Q&A]
Has anyone trained a subject face in the Dreambooth extension using any SD v2.1?

Resolution 768

This on a 3060 12gb

NOTE - NOTHING TO DO WITH LORA

**Comment by poondoggle** [comment]
Not without unchecking text encoding and that renders 2.1 useless from the tests I've done with my 3060.

**Reply by TypicalPlay** [reply]
Yes, was able to complete the training for SD 2.1 with the following setting's on a 3060 12gb

Memory Attention - xformers
Train Text Encoder  - UNCHECKED

The results were a bad resemblance of the subject's face.
But on one occasion  setting the CFG to 5 it came out brilliantly.
BUT, you cannot go about pressing the Generate button a million times to get these rare images.

-----------------------------------------------

Today was able to complete the training for SD 2.1 with the following setting's on a 3060 12gb

Memory Attention - flash_attention
Train Text Encoder  - CHECKED

ALL THE SAMPLE generated as part of training were black.

Stopped midway. It was taking a lot of time.

BUT no cuda memory errors

**Comment by zahdab** [comment]
for black images, try setting memory attention to xformers when training ... it worked for me...  i trained for 768 model using ... though the model lost a lot of detailed ... even if i dont promp the subject everything became unrefined ... l
